src/blk/kernel/KernelDevice.cc

```diff
@@ -541,6 +541,19 @@ void KernelDevice::_aio_stop()
   if (aio) {
     dout(10) << __func__ << dendl;
     aio_stop = true;
+    IOContext wakeup_ctx(cct, nullptr, false);
+    wakeup_ctx.num_running++;
+    std::list<aio_t> batch;
+    aio_t aio_op(&wakeup_ctx, fd_directs[WRITE_LIFE_NOT_SET]);
+
+    // IO_CMD_NOOP is not implemented in the Linux kernel; using IO_CMD_PREAD with a 0-byte read as a substitute.
```

Comment:
Can we use IO_CMD_PREAD for ioring mode as well? Do we really need that additional complication here?